### PR TITLE
Improve search by removing all whitespaces from fields and the query …

### PR DESCRIPTION
**Pull request by bartoszkp**
…string

### CODE REVIEW

File: house_of_refuge/frontend/src/components/ResourceList.js [hunk 1]
```diff
@@ -127,7 +125,7 @@ export const ResourceList = ({initialResources, sub, subHandler, user, clearActi
                 .filter(r => statusFilter ? statusFilter.includes(r.resource) : true)
                 .filter(r => hotTopic ? isHot(r) : true)
                 .filter(r => turtleFilter ? r.turtle : true)
-                .filter(r => searchQuery ? resourceAsString(r).search(searchQuery) > -1 : true),
+                .filter(r => searchQuery ? ResourceAsStringForSearch(r).search(searchQuery) > -1 : true),
```
Comment (jbylina) [inline house_of_refuge/frontend/src/components/ResourceList.js]: Let's fix one minor bug if we touch those lines. `search` method allows the regex pattern in the argument. If the user types `+` then the app crashes.


```suggestion
                .filter(r => searchQuery ? ResourceAsStringForSearch(r).indexOf(searchQuery) > -1 : true),
```

**Review by jbylina — CHANGES_REQUESTED** [review]
Look good 👍 Thanks for your contribution! 

In the future feel free to create a branch in `ukrainski-dom/housing-app` repository. Then I can provide review changes in code.

**Review by jbylina — CHANGES_REQUESTED** [review]
Look good 👍 Thanks for your contribution! 

In the future feel free to create a branch in `ukrainski-dom/housing-app` repository. Then I can provide review changes in code.

**Comment by jbylina** [comment]
> Look good 👍 Thanks for your contribution!
> 
> In the future feel free to create a branch in `ukrainski-dom/housing-app` repository. Then I can provide review changes in code.

Oh... I can commit changes in pull requests even when source branch is a repo on your account 🎉 @bartoszkp merged! Thanks!

**Comment by bartoszkp** [comment]
Thanks Jacek!

**Pull request merged**